supertokens_python/recipe/thirdpartypasswordless/utils.py

```diff
@@ -147,4 +172,26 @@ def validate_and_normalise_user_input(
     if get_link_domain_and_path is None:
         get_link_domain_and_path = default_get_link_domain_and_path(recipe.app_info)
 
-    return ThirdPartyPasswordlessConfig(override=OverrideConfig(functions=override.functions, apis=override.apis), providers=providers, contact_config=contact_config, flow_type=flow_type, get_link_domain_and_path=get_link_domain_and_path, get_custom_user_input_code=get_custom_user_input_code, email_verification_feature=validate_and_normalise_email_verification_config(recipe, email_verification_feature, override))
+    def get_email_delivery_config(
+        tppless_recipe: RecipeImplementation
+    ) -> EmailDeliveryConfigWithService[TypeThirdPartyPasswordlessEmailDeliveryInput]:
+        if email_delivery_config and email_delivery_config.service:
+            return EmailDeliveryConfigWithService(
+                service=email_delivery_config.service,
+                override=email_delivery_config.override
+            )
+
+        passwordless_feature = InputPasswordlessConfig(
+            contact_config.create_and_send_custom_email if contact_config.contact_method != "PHONE" else None
+        )
+        email_service = BackwardCompatibilityService(
+            recipe.app_info,
+            tppless_recipe,
+            email_verification_feature,
+            passwordless_feature
+        )
+
+        return EmailDeliveryConfigWithService(email_service, override=None)  # FIXME: Can override=None be bad?
```

Comment:
Do not pass override=None here. Pass what the user is giving..

Reply:
Fixed.